src/migrations/__tests__/migrations.test.ts

```diff
@@ -67,4 +67,17 @@ describe(`@/migrations`, () => {
 
     expect(storage.getString(throwMigrationName)).toBeFalsy();
   });
+
+  test(`migration in debug mode does not mark as complete`, async () => {
+    const throwMigrationName = 'migration_debug' as MigrationName;
+    const throwMigration: Migration = {
+      debug: true,
+      name: throwMigrationName,
+      async migrate() {},
+    };
+
+    await runMigrations([throwMigration]);
+
+    expect(storage.getString(throwMigrationName)).toBeFalsy();
+  });
```

Comment:
Renamed this file for easier recognition and finding within the codebase. Other file was required in order to safely mock the env.